asr_msgs/AsrViewport viewport
---
asr_msgs/AsrViewport filteredViewport
bool isBeenFiltered